code/game/objects/items/mr_bones.dm

```diff
@@ -0,0 +1,92 @@
+/obj/item/mr_bones_voice_box // Why not include it in anyways?
+	name = "voice box"
+	desc = "It sounds like an early era Source game."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "voicebox"
+	inhand_icon_state = "nothing"
+	mouse_opacity = 0
+	var/list/bone_phrases = list("ERROR: Incorrect Answer","Roleplay","No","Solve the Riddle","Repeat","Your Problem")
+
+/obj/item/mr_bones_voice_box/attack_self(mob/user)
+	var/choice = input(user, "Select Your Phrase", "Mr.Bones Rattler 3000") as null|anything in bone_phrases
+
+	switch(choice)
```

Comment:
this switch statement doesn't have the phrases in the same ordering as the bone_phrases list, and that mildly triggers me